lib/menubar/init.lua

```diff
@@ -150,6 +151,7 @@ local function write_count_table(count_table)
             io.write(str)
         end
         io.flush()
```

Comment:
What the heck is this code doing? Doesn't `io.output(foo)` redirect all output to that file? And the `io.flush()` seems quite obviously unnecessary. (Yes, I know that this is unrelated to this pull request)